spec/models/article_spec.rb

```diff
@@ -16,6 +16,22 @@
     it { should validate_uniqueness_of :title }
   end
 
+  describe ".published" do
+    it "returns the articles that have a published_at timestamp before the current time" do
+      published_article = create :article
+      article_published_today = create :article, published_at: 3.hours.ago
+      _unpublished_article = create :article, :unpublished
+      _article_to_be_published_today = create :article, published_at: 3.hours.from_now
+
+      expect(Article.published.ids).to eq(
```

Comment:
```suggestion
      expect(Article.published.ids).to match_array(
```

It doesn't look like this scope defines ordering, so you can't guarantee items will be in this order.

Reply:
If that's important/expected, I would recommend changing the scope to set an order.

Reply:
I think when the order becomes a concern, I'll add a separate scope for that, but this is fine for now.